core/src/test/scala/unit/kafka/server/BaseRequestTest.scala

```diff
@@ -80,14 +80,14 @@ abstract class BaseRequestTest extends KafkaServerTestHarness {
     new Socket("localhost", s.boundPort(ListenerName.forSecurityProtocol(protocol)))
   }
 
-  private def sendRequest(socket: Socket, request: Array[Byte]) {
+  def sendRequest(socket: Socket, request: Array[Byte]) {
```

Comment:
Maybe we should keep this private, rename the existing public methods to have `receive` in the name and introduce a `send` that doesn't do the receive (but still takes `ApiKeys`, etc.)